src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -59,10 +61,12 @@ def setUp(self):
             build_paths = json.load(build_vars)
         self.basepath = os.path.normpath(build_paths['PREFIX'] + "/../")
 
-        self.server_group = self.params.get("server_group", '/server/',
+        self.server_group = self.params.get("name", '/server_config/',
                                             'daos_server')
         self.daos_test = self.basepath + '/install/bin/daos_test'
         self.orterun = self.basepath + '/install/bin/orterun'
+        #To generate the seperate client log file
+        self.orterun_env = '-x D_LOG_FILE={}'.format(CLIENT_LOG)
```

Comment:
This just gets the client logging to `/tmp/client.log` on the remote node.  It still needs to be collected to the local node and added to the artifacts that Jenkins collects as specified in the `Jenkinsfile`.

Since you are reworking how the logging is being done you might as well apply the same logic
https://github.com/daos-stack/daos/blob/7144ce68f1b670db343cb1b71dabc968d467f4c5/src/tests/ftest/daos_test/daos_core_test.py#L87-L96
to the client logs that is being used to collate the server logs.  This way if you get them into the same directory on the remote node as the server log is in, the existing log collection
https://github.com/daos-stack/daos/blob/7144ce68f1b670db343cb1b71dabc968d467f4c5/ftest.sh#L258-L259
and artifacting
https://github.com/daos-stack/daos/blob/7144ce68f1b670db343cb1b71dabc968d467f4c5/Jenkinsfile#L698
process will work on them.